What the? I wonder if they really check their sources before baking a new version ...

---
diff -urpN a/include/nana/gui/detail/native_window_interface.hpp b/include/nana/gui/detail/native_window_interface.hpp
--- a/include/nana/gui/detail/native_window_interface.hpp	2016-07-19 23:47:42.039678224 +0200
+++ b/include/nana/gui/detail/native_window_interface.hpp	2017-06-06 11:50:44.154404510 +0200
@@ -15,6 +15,7 @@
 
 #include "../basis.hpp"
 #include <nana/paint/image.hpp>
+#include <functional>
 
 namespace nana
 {
